superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -442,3 +438,64 @@ export const uploadUserPerms = (
     canUploadData: canUploadCSV || canUploadColumnar || canUploadExcel,
   };
 };
+
+export function getFilterValues(
+  tab: TableTab,
+  welcomeTable: WelcomeTable,
+  user?: User,
+  otherTabFilters?: Filter[],
+): FilterValue[] {
```

Comment:
this util is a bit of a monster, but it's a compilation of the filter logic that was scattered around the subcomponents of the Welcome component.

Reply:
Thanks for centralizing the logic.